modules/apps/portal-security/portal-security-auth-test/src/testIntegration/java/com/liferay/portal/security/auth/test/AuthenticatedSessionManagerUtilTest.java

```diff
@@ -129,6 +131,15 @@ public void testRememberMeLogout() throws Exception {
 			CookiesConstants.CONSENT_TYPE_FUNCTIONAL, cookie,
 			mockHttpServletRequest, mockHttpServletResponse);
 
+		ThemeDisplay themeDisplay = ThemeDisplayFactory.create();
+
+		themeDisplay.setCompany(
+			_companyLocalService.getCompany(TestPropsValues.getCompanyId()));
+		themeDisplay.setSiteGroupId(TestPropsValues.getGroupId());
```

Comment:
@stian-sigvartsen This makes a lot more sense, I didn't know about `TestPropsValues`, thank you!